app/views/layouts/application.html.erb

```diff
@@ -8,115 +8,86 @@
-            <span class="sr-only">Toggle navigation</span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-          </button>
-        </div>
-        <a href="<%= root_path%>" class="navbar-brand" id="logo"> <%= image_tag("logo_white.png")%></a>
-        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-2">
-          <ul class="nav navbar-nav">
-            <li class="li-nav"><%= link_to "Directory", root_path, class: "directory" %></li>
-            <li class="li-nav"><%= link_to "Events", events_path, class: "directory" %></li>
-            <li class="li-nav"><%= link_to "News", news_index_path, class: "directory" %></li>
+      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-2">
+        <ul class="nav navbar-nav">
+          <li class="li-nav"><%= link_to "Directory", root_path, class: "directory" %></li>
+          <li class="li-nav"><%= link_to "Events", events_path, class: "directory" %></li>
+          <li class="li-nav"><%= link_to "News", news_index_path, class: "directory" %></li>
+          <% if user_signed_in? %>
+            <% if current_user && current_user.admin? %>
+            <li class="dropdown">
+              <a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expaned="false" href="#"><i class="fa fa-user fa-1x" aria-hidden="true"></i> Admin Panel <i class="fa fa-caret-down fa-lg" aria-hidden="true"></i></a>
+              <ul class="dropdown-menu text-center">
+                <li class="li-nav"><%= link_to "Users", admin_users_path%></li>
+                <li class="li-nav"><%= link_to "Technologies", admin_technologies_path%></li>
+                <li class="user-profile"><%= link_to "Edit Profile", edit_user_path(current_user) %></li>
+                <li class="user-profile"><%= link_to "Edit Password", edit_password_change_path(current_user) %></li>
+                <li class="divider"></li>
+                <li class="user-profile"><%= link_to "Log out", sessions_path, method: :delete %></li>
+                <%end%>
+                <% else %>
+                <li class="li-nav"><%= link_to "Sign In", new_session_path %></li>
+                <li class="li-nav"> <%= link_to "Sign Up", new_user_path, id: "btn-sign-up" %></li>
+                <% end %>
+              </ul>
+            </li>
           </ul>
         </div>
       </div>
     </nav>
-  <% end %>
-
-  <% if flash[:notice] %>
+    <% if flash[:notice] %>
     <div class="alert alert-success">
-      <%= flash[:notice] %>
-    </div>
-  <% elsif flash[:alert] %>
-    <div class="alert alert-danger">
-      <%= flash[:alert] %>
-    </div>
-  <% end %>
-
-  <div class="container">
-    <%= yield %>
+    <%= flash[:notice] %>
   </div>
+  <% elsif flash[:alert] %>
+  <div class="alert alert-danger">
```

Comment:
yes, I has been closed. it is not created by us but what it is doing is to wrap the alert or notice inside a div with bootstrap class